website/docs/jellyfish/api/poolpair.md

```diff
@@ -161,6 +161,29 @@ interface PoolShareOptions {
 }
 ```
 
+## removePoolLiquidity
+
+Remove pool liquidity transaction
+
+```ts title="client.poolpair.addPoolLiquidity()"
```

Comment:
This is not addPoolLiquidity. Should be removePoolLiquidity

Reply:
Thanks for noticing! I'll make the changes.